test/unit/chats_test.ts

```diff
@@ -138,6 +138,41 @@ describe('sendMessage valid response', () => {
   });
 });
 
+describe('sendMessage subsequent calls', () => {
+  it('sendMessage errors should not persist', async () => {
+    const client = new GoogleGenAI({vertexai: false, apiKey: 'fake-api-key'});
+    const chat = client.chats.create({model: 'gemini-1.5-flash'});
+
+    const modelsModule = client.models;
+    spyOn(modelsModule, 'generateContent').and.returnValue(
+      Promise.reject('test error'),
+    );
+
+    const message1 = chat.sendMessage({message: 'send message 1'});
+    await expectAsync(message1).toBeRejectedWith('test error');
```

Comment:
Could you move the assert to the bottom? the repo follows the arrange-act-assert pattern for tests structure.